AssetRipper.Library/Exporters/Shaders/UnityPatch/YamlShaderPostprocessor.txt

```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+using UnityEditor;
+
+namespace AssetRipperPatches.Editor
+{
+    /// <summary>
+    /// This script is AssetRipper's patch for shaders exported as YAML assets.
+    /// Such a shader can be assigned to and used by a material as a regular .shader asset,
+    /// but it does not work with Shader.Find() unless we explicitly register it. 
+    /// </summary>
+    public class AssetRipperShaderPostprocessor : AssetPostprocessor
+    {
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
+        {
+            foreach (var importedAsset in importedAssets)
+            {
+                if (!importedAsset.EndsWith(".asset")) continue;
```

Comment:
If Unity allows Ordinal string comparisons, that method overload should be used.

Reply:
Yes, even Unity 5 has string comparison overloads. I think OrdinalIgnoreCase should be used here.

Reply:
Not ignore case, just Ordinal